src/Shellbar/Shellbar.js

```diff
@@ -121,6 +122,12 @@ class Shellbar extends Component {
         return (
             <div className={shellbarClasses}>
                 <div className='fd-shellbar__group fd-shellbar__group--product'>
+                    {backAction && <Button
+                        className='fd-shellbar__button'
```

Comment:
We should add an aria-label here for "Back" - see the `localizedText` prop below and copy what's done with `counterLabel`. 

Also, `npm run test:visual:update` should get you squared away with travis! 🎉